space-plugins/nuxt-base/server/utils/result.ts

```diff
@@ -0,0 +1,46 @@
+import { safeParse } from 'valibot';
+import {
+	ResultSchema,
+	type FailureResult,
+	type Result,
+	type SuccessResult,
+} from '../../types';
```

Comment:
It didn't work using the alias approach `~/types` that is the reason I used relative path here.

![image](https://github.com/storyblok/space-tool-plugins/assets/1240591/9b5ddd2f-439c-48be-8ae5-18fc6aa48bc9)

https://nuxt.com/docs/guide/going-further/layers#relative-paths-and-aliases